spec/lib/task-data/schemas/install-coreos-spec.js

```diff
@@ -16,7 +16,8 @@ describe(require('path').basename(__filename), function() {
         "installDisk": "/dev/sda",
         "comport": "ttyS0",
         "osType": "linux",
-        "rootPassword": "RackHD"
+        "rootPassword": "RackHD",
+        "ignitionScriptUri": "generic.json"
```

Comment:
@yyscamper, I purposefully excluded any ignition script, not wanting to customize CoreOS unless someone overrides `ignitionScriptUri`.
As currently written, Ignition will silently fail, indicating in the log that `generic.json` could not be found. If it's preferable, I can put a dummy ignition script in that does something like creates a `README` or something similar.